### PR TITLE
Move dynamic channel types into a type module within their package

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

**Comment by nicpottier** [comment]
Aren't you missing the 🌩 emoji in this PR? :)

File: temba/channels/types/external/__init__.py [hunk 1]
```diff
@@ -1,77 +1,3 @@
...
-
-        except Exception as e:
-            raise SendException(six.text_type(e), event=event, start=start)
-
-        if response.status_code != 200 and response.status_code != 201 and response.status_code != 202:
-            raise SendException("Got non-200 response [%d] from API" % response.status_code,
-                                event=event, start=start)
-
-        Channel.success(channel, msg, WIRED, start, event=event)
+from .type import ExternalType  # noqa
```
Comment (nicpottier) [inline temba/channels/types/external/__init__.py]: Just out of curiosity, why do these need the noqa?

Comment (rowanseymour) [inline temba/channels/types/external/__init__.py]: Because flake8 doesn't know that we import the class dynamically from this package

Comment (rowanseymour) [inline temba/channels/types/external/__init__.py]: i.e. thinks its un unused import

Comment (nicpottier) [inline temba/channels/types/external/__init__.py]: Oh funny, I ran into the same thing when trying to import the courier stuff into .models import my PR

**Pull request merged**